pkg/ReleaseNoteService.go

```diff
@@ -182,10 +185,20 @@ func (impl *ReleaseNoteServiceImpl) GetReleases() ([]*common.Release, error) {
 			return releaseList, fmt.Errorf("failed operation on fetching releases from github, attempted 3 times")
 		}
 	}
-
+	impl.buildReleaseResponse(releaseList)
 	return releaseList, nil
 }
 
+func (impl *ReleaseNoteServiceImpl) buildReleaseResponse(releaseList []*common.Release) []*common.Release {
+	for _, release := range releaseList {
+		if strings.Contains(release.Body, PrerequisitesMatcher) {
```

Comment:
loop can be avoided here. can we use this logic while adding in list. line no. 175